<a href="https://colab.research.google.com/github/AdityaDhuri/Assignment-Questions/blob/main/Assi_3_Q1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

## A F&B manager wants to determine whether there is any significant difference in the diameter of the cutlet between two units. A randomly selected sample of cutlets was collected from both units and measured? Analyze the data and draw inferences at 5% significance level. Please state the assumptions and tests that you carried out to check validity of the assumptions.



In [ ]:
import pandas as pd
import numpy as np
from scipy import stats
from scipy.stats import norm

In [ ]:
data=pd.read_csv('A:\ExcelR Assi\Assi3...Hypothesis testing\Cutlets.csv')

In [ ]:
data.head()

Unit A  Unit B
0  6.8090  6.7703
1  6.4376  7.5093
2  6.9157  6.7300
3  7.3012  6.7878
4  7.4488  7.1522

Assume Null hypothesis (Ho): There is no difference in diameter of cutlets between two units
Alternate hypothesis (Ha): There is significance difference in diameter of cutlets between two units
2 sample 2 tail test

In [ ]:
unitB = pd.Series(data.iloc[:,1])
unitB

0     6.7703
1     7.5093
2     6.7300
3     6.7878
4     7.1522
5     6.8110
6     7.2212
7     6.6606
8     7.2402
9     7.0503
10    6.8810
11    7.4059
12    6.7652
13    6.0380
14    7.1581
15    7.0240
16    6.6672
17    7.4314
18    7.3070
19    6.7478
20    6.8889
21    7.4220
22    6.5217
23    7.1688
24    6.7594
25    6.9399
26    7.0133
27    6.9182
28    6.3346
29    7.5459
30    7.0992
31    7.1180
32    6.6965
33    6.5780
34    7.3875
Name: Unit B, dtype: float64

In [ ]:
p_value = stats.ttest_ind(unitA,unitB)
p_value

Ttest_indResult(statistic=0.7228688704678063, pvalue=0.4722394724599501)

In [ ]:
p_value[1] #2-tail probability

0.4722394724599501

In [ ]:
#comparing p_value with given significance level of 5%


Inference: As p_value (0.4722)  < 0.05. Thus there is no difference in diameter of cutlets between two units